src/react/src/util/util.facilityClaimsXLSX.js

```diff
@@ -18,8 +19,11 @@ const formatFacilityClaimsDataForXLSX = facilityClaims =>
             facilityClaim.contributor_name,
             facilityClaim.facility_country_name,
             moment(facilityClaim.created_at).format('LL'),
-            moment(facilityClaim.updated_at).format('LL'),
+            facilityClaim.claim_decision !== null
+                ? moment(facilityClaim.claim_decision).format('LL')
+                : 'N/A',
```

Comment:
Maybe it is better to replace this 'N/A' into a const and import it? We use it in `src/react/src/components/DashboardClaimsListTable.jsx` as well. What if client decide to change it to 'None' or simple '-' later?

Reply:
Fixed.